### PR TITLE
blog/replicating-fama-french-5-factor-model

### PR DESCRIPTION
**Pull request by christophscheuch**
In order to replicate the 5 FF factors, I had to do the following:

- Extend Compustat query & variable construction
- Change Fama-French data download
- Adapt the whole replicating-fama-and-french-factors chapter

I currently only did the last part in R because I want to get approval first. 

The following parts still need to be done:

- Adapt python/replicating-fama-and-french-factors
- Update r/changelog

### CODE REVIEW

File: blog/fama-french-three-vs-five-factors/index.qmd [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+date: "2023-09-20"
+description: An explanation for the difference in the value factors of FF3 and FF5 data
+image: thumbnail.png
+image-alt: Two stylized high rise office buildings, one small building and one large building, both buildings visible in the frame. The background has a futuristic setting, with a backdrop of a digital, grid-like landscape symbolizing the internet. The color palette should be a combination of cool blues and warm yellow. Created with DALL-E 2.
+categories: 
+  - Data
+  - Replication
+--- 
+
+In our book chapter [Replicating Fama and French Factors](../../r/replicating-fama-and-french-factors), we show how to construct factor portfolios that are fairly close to the popular data from [Prof. Kenneth French finance data library](https://mba.tuck.dartmouth.edu/pages/faculty/ken.french/data_library.html). In this blog post, I want to elaborate a bit more on the subtle differences between the size and value data in the Fama-French three (FF3) and five (FF5) factor data.
```
Comment (patrick-weiss) [inline blog/fama-french-three-vs-five-factors/index.qmd:17]: Maybe add the two paper citations as footnotes.

Would it make sense to investigate Google search requests for the differences between the factors for SEO?

**Review by patrick-weiss — CHANGES_REQUESTED** [review]
Some small suggestions and a potential extension. Great post overall.

File: blog/fama-french-three-vs-five-factors/index.qmd [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+```{r}
+model_smb <- lm(smb_ff3 ~ smb_ff5, data = factors_ff)
+summary(model_smb)
+```
+
+Regressing the FF3 SMB factor on its FF5 counterpart yields a coefficient of `r round(model_smb$coefficients[2], 2)` and an R-squared around `r round(summary(model_smb)$adj.r.squared, 2) * 100`%, so definitely no perfect co-movement. Where does this difference come from?
+
+There are actually two explanations for the differences:
+
+1. The value portfolios `portfolio_bm` are constructed as independent sorts in FF3, while they are the result of dependent sorts in FF5 (depending on `portfolio_size`).
```
[the remark this answers is not in the document]
Comment (christophscheuch) [reply blog/fama-french-three-vs-five-factors/index.qmd:79]: Yeah, I mixed up size and value while I wrote the blog post. The second argument stands, but the first one should go, right?

[the remark this answers is not in the document]
Comment (christophscheuch) [reply blog/fama-french-three-vs-five-factors/index.qmd:79]: I moved the second argument to the top and adjusted the other argument.

File: blog/fama-french-three-vs-five-factors/index.qmd (hunk 1 above)
Comment (christophscheuch) [reply blog/fama-french-three-vs-five-factors/index.qmd:18]: We hardly get any impressions for fama french replication, so I'd not overinvest for now.

**Comment by christophscheuch** [comment]
From my side the PR is ready to merge, waiting for approval from @patrick-weiss. I'll update changelog after merge and reference the PR.

**Pull request merged**